### PR TITLE
Skip fixtures feature introduced

### PR DESCRIPTION
**Pull request by SabaunT**
This hot fix is introduced until we decide finally on having salt in `create_program_with_gas` function. If salt remains, then the solution in #739 will be merged. Otherwise, we take salt management responsibility and introduce it in https://github.com/gear-tech/gear/issues/703

### CODE REVIEW

File: gear-test/spec/test_create_program.yaml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 title: Test create_program sys-call
 
+must_skip: true
```
Comment (shamilsan) [inline gear-test/spec/test_create_program.yaml:3]: Maybe just `skip`?
```suggestion
skip: true
```

**Review by shamilsan — APPROVED** [review]
Good but I'd rename the field.

**Review by ukint-vs — APPROVED** [review]

File: gear-test/src/check.rs [hunk 2]
```diff
@@ -668,7 +671,7 @@ where
                         skip_memory,
                     )
                 };
-                if output != "Ok".bright_green() {
+                if !(output.starts_with("Ok") | output.starts_with("Skipped")) {
```
Comment (SabaunT) [inline gear-test/src/check.rs:674]: Should be `||`. Sorry for that.

File: gear-test/spec/test_create_program.yaml (hunk 1 above)
Comment (SabaunT) [reply gear-test/spec/test_create_program.yaml]: done

**Pull request merged**